src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -1,10 +1,6 @@
 package seedu.addressbook.commands;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
While it appears more efficient to group all the removed imports under `java.util.*`, imported classes should always be listed explicitly as mentioned in our [coding standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html).